### PR TITLE
Update flight_tasks.md

### PR DESCRIPTION
**Pull request by alnI3S**
Add missing steps to test new Flight Task

### CODE REVIEW

**Comment by hamishwillee** [comment]
@MaEtUgR Can you please review this.

File: en/concept/flight_tasks.md [hunk 1]
```diff
@@ -118,9 +123,10 @@ The instructions below might be used to create a task named _MyTask_:
       * @value 0 Direct velocity
       * @value 3 Smoothed velocity
       * @value 4 Acceleration based
+      * @value 5 Acceleration based
```
Comment (MaEtUgR) [inline en/concept/flight_tasks.md:126]: You'd probably want to call it something like `My Task` for the drop down menu in the UI to select

**Review by MaEtUgR — APPROVED** [review]
Thanks for correcting the steps while following the guide, this is really helpful 👍🏻👍🏻

**Comment by hamishwillee** [comment]
@MaEtUgR Once @alnI3S has looked at your suggestion feel free to merge. I'll be back next week if you don't find time.

File: en/concept/flight_tasks.md (hunk 1 above)
Comment (hamishwillee) [reply en/concept/flight_tasks.md:126]: Let's just do this.

```suggestion
      * @value 5 My task
```

**Pull request merged**